students/saleta_sebastian/lesson_11_argparse_and_math/regex_search.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import os
+import sys
+import re
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Script to find sentence'
+                                                 ' in txt files ')
+    parser.add_argument('-p', '--sentence',
```

Comment:
Shouldn't be ```parser.add_argument('-s', '--sentence',```

As I know you from this course you would like to fix this, am I wrong?

Reply:
I love mistakes like this 👍